### PR TITLE
added automatic cutter workplane tests

### LINKED ISSUE #395: Test for wedge cut in different workplanes

### PR DESCRIPTION
**Pull request by billingsley-john**
## Proposed changes

PR which adds tests to check that the automatic wedge cutter for tf coils works in different workplanes. Partially resolves #395.
Workplane error, as described in #308, prevents full resolution as feature not fully working in all workplanes.

## Types of changes

What types of changes does your code introduce to the Paramak?
_Put an `x` in the boxes that apply_

- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Code refactoring
- [ ] Documentation Update (if none of the other choices apply)
- [x] New tests

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [ ] Pep8 applied
- [x] Unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

## Further comments

No further comments

### CODE REVIEW

File: tests/test_parametric_components/test_ToroidalFieldCoilTripleArc.py [hunk 1]
```diff
@@ -32,8 +32,21 @@ def test_ToroidalFieldCoilTripleArc_rotation_angle(self):
...
         test_shape.rotation_angle = 180
+        assert test_shape.volume == pytest.approx(test_volume * 0.5, rel=0.01)
 
+        test_shape.rotation_angle = 360
+        test_shape.workplane = "YZ"
+        test_volume = test_shape.volume
+        test_shape.rotation_angle = 180
         assert test_shape.volume == pytest.approx(test_volume * 0.5, rel=0.01)
+
+        # test_shape.rotation_angle = 360
```
Comment (Shimwell) [inline tests/test_parametric_components/test_ToroidalFieldCoilTripleArc.py:48]: are these commented out lines needed

File: tests/test_parametric_components/test_ToroidalFieldCoilCoatHanger.py [hunk 2]
```diff
@@ -35,8 +35,21 @@ def test_ToroidalFieldCoilCoatHanger_rotation_angle(self):
...
         test_shape.rotation_angle = 180
+        assert test_shape.volume == pytest.approx(test_volume * 0.5, rel=0.01)
 
+        test_shape.rotation_angle = 360
+        test_shape.workplane = "YZ"
+        test_volume = test_shape.volume
+        test_shape.rotation_angle = 180
         assert test_shape.volume == pytest.approx(test_volume * 0.5, rel=0.01)
+
+        # test_shape.rotation_angle = 360
```
Comment (Shimwell) [inline tests/test_parametric_components/test_ToroidalFieldCoilCoatHanger.py:51]: ares these commented out lines needed

File: tests/test_parametric_components/test_ToroidalFieldCoilPrincetonD.py [hunk 3]
```diff
@@ -43,8 +43,21 @@ def test_ToroidalFieldCoilPrincetonD_rotation_angle(self):
...
         test_shape.rotation_angle = 180
+        assert test_shape.volume == pytest.approx(test_volume * 0.5, rel=0.01)
 
+        test_shape.rotation_angle = 360
+        test_shape.workplane = "YZ"
+        test_volume = test_shape.volume
+        test_shape.rotation_angle = 180
         assert test_shape.volume == pytest.approx(test_volume * 0.5, rel=0.01)
+
+        # test_shape.rotation_angle = 360
```
Comment (Shimwell) [inline tests/test_parametric_components/test_ToroidalFieldCoilPrincetonD.py:59]: are these commented out lines needed

File: tests/test_parametric_components/test_ToroidalFieldCoilTripleArc.py (hunk 1 above)
Comment (billingsley-john) [reply tests/test_parametric_components/test_ToroidalFieldCoilTripleArc.py:48]: These commented out tests are required, but will only pass when #308 is resolved

File: tests/test_parametric_components/test_ToroidalFieldCoilCoatHanger.py (hunk 2 above)
Comment (billingsley-john) [reply tests/test_parametric_components/test_ToroidalFieldCoilCoatHanger.py:51]: See above

File: tests/test_parametric_components/test_ToroidalFieldCoilPrincetonD.py (hunk 3 above)
Comment (billingsley-john) [reply tests/test_parametric_components/test_ToroidalFieldCoilPrincetonD.py:59]: See above

**Pull request merged**